assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -5639,6 +5640,7 @@ public void doSave_toggle_remove_review(RunData data) {
                 }else{
                     submissionId = submissionRef.substring(i + 1);
                 }
+                assignmentRef = submissionRef.substring(0, submissionRef.length()-submissionId.length()-1);
```

Comment:
AssignmentReferenceReckoner should be used here.

Reply:
@ern Now, I'm using AssignmentReferenceReckoner. Please, take a look!. Thanks